devops/Terraform/backend.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "azurerm" {
+    resource_group_name  = "obelisk-infra-rg"
+    storage_account_name = "obeliskinfrastorage"
+    container_name       = "obelisk-frontend"
```

Comment:
@PeterBeyond we're not planning to push both Obelisk SF FE and Obelisk Vuetify FE to an Azure env? If we are, please let me know, I'll need to change those settings. This is the name of the container in Azure Storage where Terraform stored the state of the current deployment infrastructure. We'll need sep containers for SF anf VUetify FE.